docs/howto/enumerations.md

```diff
@@ -0,0 +1,100 @@
+# How to specify enumerations
+
+Enumerations are **finite** sets of values, which may be **named** and may
+have an associated **semantic meaning**.
+
+**Note**
+While enumerations may be used to define [classes](/definitions/classes) or [labels](/definitions/labels), they don't mecessarily share the same meaning, check the definitions.
+
+Although often being of `dataType` [`sc:Text`](http://schema.org/Text), enumerations can also be of other `dataType`s (e.g. `sc:Integer`).
+
+For example, the titanic dataset uses an enumeration to specify
+that the passengers gender field, of type `sc:Text`, can only take two values: `male` or `female`.
+
+## Declaring enumerations
+
+There are two ways to declare enumerations:
+
+1. Directly on a `ml:Field`. This is a simple way to define an enumeration
+   but has some limitations discussed below.
+2. Using a dedicated `ml:RecordSet`, holding the enumeration values. Such
+   definition allows enumerations to be fully defined, with a name and a
+   semantic URL.
+
+### `ml:Field` enumeration
+
+In the following example, extracted from the [coco2014 Croissant definition](https://github.com/mlcommons/croissant/blob/main/datasets/coco2014/metadata.json), the field `supercategory` is declared to be
+an enumeration using the `"ml:isEnumeration": true` property. This means that
+there is a finite number of `supercategory` values, and that it is expected
+that many rows will have the same value.
```

Comment:
For me, this text would be more easily readable if the example was a little more elaborated.
Something like "In the instances_val2014.json, for example, coco2014 describes 80 possible categories. Each category has an identifier, a name (e.g.: "person", "bicycle" or "car") and a supercategory (e.g. "person" or "vehicle").

Reply:
+1